layers/+lang/scheme/packages.el

```diff
@@ -102,17 +102,44 @@
         "sR" 'geiser-eval-region-and-go
         "ss" 'geiser-set-scheme)
 
+      (evil-define-key 'insert 'geiser-repl-mode-map
+        (kbd "S-<return>") 'geiser-repl--newline-and-indent)
+
       (evil-define-key 'normal 'geiser-repl-mode-map
-      "gj" 'geiser-repl-next-prompt
-      "gk" 'geiser-repl-previous-prompt)
+        "]]" 'geiser-repl-next-prompt
+        "[[" 'geiser-repl-previous-prompt
+        "gj" 'geiser-repl-next-prompt
+        "gk" 'geiser-repl-previous-prompt)
 
+      (spacemacs/declare-prefix-for-mode 'geiser-repl-mode "mh" "help")
       (spacemacs/set-leader-keys-for-major-mode 'geiser-repl-mode
-        "q" 'geiser-repl-exit
+        "b" 'geiser-pop-symbol-stack
         "c" 'geiser-repl-clear-buffer
         "i" 'geiser-repl-interrupt
+        "f" 'geiser-load-file
+        "l" 'geiser-insert-lambda
         "m" 'geiser-repl-import-module
         "u" 'geiser-repl-unload-function
-        "h" 'geiser-doc-symbol-at-point))))
+        "hd" 'geiser-doc-symbol-at-point
+        "s" 'geiser-squarify
+        "q" 'geiser-repl-exit
+        "hc" 'geiser-customize-implementation)
+
+      (evilified-state-evilify-map geiser-doc-mode-map
+        :mode geiser-doc-mode
+        :eval-after-load geiser-doc
+        :bindings
+        "o" 'link-hint-open-link
+
+        "]]" 'geiser-doc-next-section
+        "[[" 'geiser-doc-previous-section
+        ">" 'geiser-doc-next
+        "<" 'geiser-doc-previous
+
+        "gb" 'geiser-doc-previous
```

Comment:
for "gb" and "gf"

do you mean backward and forward?

normally we just use "gn" and "gp" as next and previous.